cmd/drone-downstream/main.go

```diff
@@ -23,7 +23,9 @@ func main() {
 	settings := &plugin.Settings{}
 
 	if _, err := os.Stat("/run/drone/env"); err == nil {
-		godotenv.Overload("/run/drone/env")
+		if err := godotenv.Overload("/run/drone/env"); err != nil {
+			errors.HandleExit(err)
+		}
```

Comment:
The `main.go` file is generated from https://github.com/drone-plugins/boilr-plugin/blob/master/template/cmd/%7B%7BExecutable%7D%7D/main.go so you shouldn't be modifying this file at all.

@tboerger is this a change we should make to the boilr source?

Reply:
IMHO we don't need to care about that error

Reply:
OK, will remove thsi.